app/views/templates/Layout.scala.html

```diff
@@ -64,10 +64,6 @@
     )
 }
 
-@additionalScripts = {
```

Comment:
the script `javascripts/application.min.js` does not exist - it's throwing 404 responses so I'm removing it here